src/App.css

```diff
@@ -1,38 +1,303 @@
 .App {
   text-align: center;
+  position: relative;
+  height: 100%;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+/* Header */
+header {
+  background: #0D3B66;
```

Comment:
That was a stretch goal of mine. Since we weren't shown it directly, I didn't want to attempt it at first. Sounds like you took on Sass while I took on FontAwesome!